modules/tinymce/src/plugins/media/test/ts/browser/PlaceholderTest.ts

```diff
@@ -12,7 +12,8 @@ describe('browser.tinymce.plugins.media.core.PlaceholderTest', () => {
   const baseSettings = {
     plugins: [ 'media' ],
     toolbar: 'media',
-    extended_valid_elements: 'script[src|type]'
+    extended_valid_elements: 'script[src|type]',
+    sandbox_iframes_exclusions: []
```

Comment:
Should we also have a dedicated test that checks all the hard coded whitelisted items?

Reply:
That will be done as part of [TINY-10535](https://ephocks.atlassian.net/browse/TINY-10535)